<!--StartFragment-->


# **Summary of Whole-Genome Sequencing Accurately Identifies Resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens**

Authors (@slack): Chijioke C. Uhegwu (@Chijioke001), Merna Raafat Salem (@MernaSalem28), Olorunfemi Omobolade (@Olorunfemi), Yomna Khaled (@yomna98)

Video Presentation: <https://shorturl.at/m4yX2>


## **Introduction**

AMR poses significant challenges, especially in neutropenic cancer patients vulnerable to bacterial infections. Traditional phenotypic AMR detection takes 48–72 hours, prompting the need for rapid molecular diagnostics (RMDs). Studies show RMDs, including WGS, effectively detect AMR in key pathogens, potentially improving timely antibiotic therapy for high-risk patients.


## **Methods**

### **Clinical Analysis**

- Specimen selection: Bacteria that caused bloodstream infections with fever and neutropenia were stored as _glycerol stocks at –80°C_ .

- Antimicrobial susceptibility testing: evaluate the resistance to four-β-lactam antibiotics.


### **WGS and Analysis**

Data: selected gram-negative bacteria species

Customized AMR protein sequences database built by merging antibiotic resistance database, comprehensive antibiotic resistance database, including β-lactamase alleles or mutations leading to β-lactam resistance.

[![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXc-horqvZ6nO4o808pmiaqgR6ynajd_jt_SK5bPE9zls_e99Xjgp3VZDZGxsi_ifxnk6hNdBPPkcKzLCLYLcJbMdtLrcyg4S_EIbJQIqBoD4Cq_dDAjBP_s0k-qCc4i-Zbf5e_TiF_P1EI-1owKMnxD7AI?key=Hg3N05G8RzqwOoq2tLdguw)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom\&p=PMC3\&id=5850535_cix41701.jpg)

**Figure 1.** Diagram showing the conversion of phenotypic predictions from genotypic detection of AMR mechanisms. For the four species under study, AMR was projected as blue bars. A mechanism unique to Escherichia coli is shown by a green bar. A process unique to _Klebsiella pneumoniae_ is shown by an orange bar. A _Pseudomonas aeruginosa_-specific process is shown by a magenta bar. An _Enterobacter cloacae_-specific pathway is shown by a purple bar. Abbreviation: P/T, piperacillin-tazobactam.


### **Statistical Analyses using Stata v-14**

- Sensitivity, Specificity, (PPV), (NPV).

- Cohen's Kappa.

- McNemar.

- Marginal Regression Model.


## **Results and Discussion** 

### **Isolate Identification and Characterization**

A total of 737 cases of monomicrobial gram-negative bacteremia were identified, with _E. coli_ (38% of 280 cases), _P-aeruginosa_ (19% of 138), _K-pneumoniae_ (14% of 102), and _Enterobacter-cloacae_ (5% of 39) cumulatively accounting for 76% of cases. 90-isolate representative samples underwent WGS, revealing genetic diversity, except for 12 _E.coli_ strains grouped in the extended-spectrum β-lactamase–producing sequence type 131.


### **β-Lactam Resistance Phenotypic Analysis** 

After combining species with β-lactams, various degrees of resistance were observed, except for E. cloacae, which showed no resistance to meropenem. Moreover, different MICs were detected for each species, highlighting their heterogeneity, which allows better assessment of WGS in predicting AMR.


### **WGS Characterization of Predicted AMR** 

133 predicted AMR events across 90 strains were identified, with mechanisms including exogenous β-lactamase acquisition (65%) and chromosomal gene inactivation, varying by species, particularly in _P. aeruginosa_ and _E. cloacae_.


### **Evaluation of WG Data for Phenotypic AMR Prediction**

In most cases, WGS predictions agreed with phenotypic susceptibility determined by BMD, with sensitivity (0.87) , specificity (0.98). Except for _E. cloacae_, the differences resulted from the inability of determining a genotypic mechanism underlying phenotypic resistance to piperacillin-tazobactam. 

![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXex90kjvTOj8ojjDevI8hcMf103pXRYw64LOxhE5JYiNu6rOZ-rucxv0fu-5z2srK6BJDIidVfpCPuQAheZnxIhcP3CSZqPFRih_sa5si3wRvs_WoRzSKc_5ZIWwt0sMdkr_YWBwwA6mk5O7myhZKxDzjit?key=Hg3N05G8RzqwOoq2tLdguw)

**Figure 2.**  Sensitivity and specificity of genotypic predictions for A-B: the four bacterial species studied, C-D: the four antibiotics studied, and E-F: the combinations of the bacterial species and the antibiotics, using different gold standards.


### **BMD With Clinical Microbiology Laboratory Data**

In AMR categorization for studied β-lactams, WGS method performed at least as the commercial methods data, in some characteristics, noticeably better.


## **Conclusion**

Using WGS, in neutropenia, compared to PCR methods, had higher specificity and sensitivity, especially in pathogens susceptible to β-lactams, identifying more AMR mechanisms, such as mutations, which made it more comprehensive and clinically suitable. The limitations include that only four β-lactams were investigated, culminating the need for further research and analysis in clinical settings.


## **Reference**

Shelburne, S. A., Kim, J., Munita, J. M., Sahasrabhojane, P., Shields, R. K., Press, E. G., Li, X., Arias, C. A., Cantarel, B., Jiang, Y., Kim, M. S., Aitken, S. L., & Greenberg, D. E. (2017). Whole-Genome Sequencing Accurately Identifies Resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens. _Clinical Infectious Diseases_, _65_(5), 738–745. <https://doi.org/10.1093/cid/cix417>

<!--EndFragment-->
